### DISCUSSION TITLE
How to jump out of the loop in visual programming

### DISCUSSION THREAD

**Discussion by szuzmh** [gopro/labs#1025] [Q&A] [answered]
![2024-08-16_155908](https://github.com/user-attachments/assets/c9403557-c5ac-49f1-ba36-ef4473e34786)

**Comment by dnewman-gpsw** [comment, marked as answer]
With conditionals.

![image](https://github.com/user-attachments/assets/965a3f33-9ba1-4bae-9537-8ba25b387ede)

In the example I using the loop count status to make the goto conditional. This will loop 10 time.

**Reply by szuzmh** [reply]
这样的话循环后就不能再执行了，还有方法能退出循环后再加任务去执行吗？

**Reply by dnewman-gpsw** [reply]
It is programming language, set any conditions you would like.  The above was simply an example.
